patches/kaggle_bigquery.R

```diff
@@ -0,0 +1,43 @@
+# This library adds support for BigQuery (via the bigrquery library), by using

```

Comment:
I'm insufficiently fluent in R to provide a good review for the use of the language.  Not sure if Vincent is comfortable with it.  Otherwise can you include someone who is?  Rachael perhaps.